### ISSUE TITLE
Tangram doesn't understand absolute URL paths

### ISSUE THREAD

**Issue by migurski** [tangrams/tangram#477]
**TO REPRODUCE THE ISSUE, FOLLOW THESE STEPS:**

Sample scene file excerpt, hosted from http://127.0.0.1:5000/scene.yaml:

```YAML
import: https://mapzen.com/carto/refill-style/4/refill-style.yaml

sources:
    _dots:
        type: GeoJSON
        url: "/{z}/{x}/{y}.json"
        max_zoom: 14
```

**RESULT:**

Tangram makes failing requests to URLs like "http://127.0.0.1:5000//12/656/1582.json" (note the double slash before the zoom level).

**EXPECTED RESULT:**

A URL starting with a slash should resolve to the root of the server, like "http://127.0.0.1:5000/12/656/1582.json".

**ENVIRONMENT:**

Tangram v0.11.6, Safari Version 9.1 (11601.5.17.1), OS X 10.11.4 (15E65). The tiles and scene file are both served with Flask, which has a `url_for()` function that produces slash-prefixed URLs.

**Comment by nvkelso** [comment]
Is this in Tangram Play or a local dev rig?

**Comment by migurski** [comment]
Local development on a website, not in Play.

**Comment by bcamper** [comment]
Thanks, this is likely a regression from when we added .zip bundles.

**Comment by bcamper** [comment]
Should be fixed with 893782bff46422d7fd400fd12cf0428f037e97b8.

@migurski I tried a similar setup to above with a local tile but if you have a a moment to verify let me know, using:
https://precog.mapzen.com/tangrams/tangram/master/dist/tangram.debug.js

**Comment by migurski** [comment]
I verified that this works with a daring double-Precog turducken.

**Comment by bcamper** [comment]
I think you need a third precog stuffed in there.

**Comment by bcamper** [comment]
This is queued for v0.11.7.

**Comment by bcamper** [comment]
Re-opening until the fix is in a public release (per standard Tangram process).

**Comment by bcamper** [comment]
Released in [v0.11.7](https://github.com/tangrams/tangram/releases/tag/v0.11.7)
